lib/ansible/plugins/filter/core.py

```diff
@@ -530,6 +530,7 @@ def filters(self):
             'basename': partial(unicode_wrap, os.path.basename),
             'dirname': partial(unicode_wrap, os.path.dirname),
             'expanduser': partial(unicode_wrap, os.path.expanduser),
+            'expandvars': partial(unicode_wrap, os.path.expandvars),
```

Comment:
In the docs, i would make note that this expands 'LOCAL' vars and not remote ones, so users realize that using remote paths with this can lead to errors.